recipes/pyopengl/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set version = "3.1.0" %}
+
+package:
+  name: pyopengl
+  version: {{ version }}
+
+source:
+  fn: PyOpenGL-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/source/P/PyOpenGL/PyOpenGL-{{ version }}.tar.gz
+  md5: 0de021941018d46d91e5a8c11c071693
+
+build:
+  script: python setup.py install
```

Comment:
Could we please add these options `--single-version-externally-managed --record record.txt`? It appears this will try to use `setuptools` so we need to stop `setuptools` from doing certain things.